tensorflow/python/lib/core/py_func.cc

```diff
@@ -39,7 +47,7 @@ PyObject* GetPyTrampoline() {
 }
 
 // Module initialization (mainly import numpy) if needed.
-void InitIfNeeded() {
+NUMPY_IMPORT_ARRAY_RETURN_TYPE InitIfNeeded() {
```

Comment:
I don't understand this change. why InitIfNeeded()'s return type has anything to do w/ import_array()'s? If NUMPY_IMPORT_ARRAY_RETURN_TYPE is defined to int, the code should not compile under gcc w/ -Wreturn-type on.